utils/api/configurations.js

```diff
@@ -63,20 +63,19 @@ export const configureRequests = ({ data, path }) => {
 // takes an array of errors for each page or component
 export const configureErrors = (errors) => {
   const env = process.env.NODE_ENV
-  let errorText = []
-  let errorTitle = ''
   const remainingErrors = errors.filter(error => error)
-  
+  let body = []
+  let title = ''
+
   if (env === 'development') {
     remainingErrors.map(error => {
-      errorText.push(JSON.stringify(error))
-      errorTitle = remainingErrors.length > 1 ? 'There were multiple errors.' : error.name
-      return { errorText, errorTitle }
+      body.push(JSON.stringify(error, null, 2))
```

Comment:
the extra parameters here help with the `pre` formatting in the component. means we'll use 2 spaces for indentation.